lib/pointing_party_web/live/card_live.ex

```diff
@@ -0,0 +1,114 @@
+defmodule PointingPartyWeb.CardLive do
+  use Phoenix.LiveView
+
+  alias PointingParty.{Card, VoteCalculator}
+  alias PointingPartyWeb.{Endpoint, Presence}
+
+  def render(assigns) do
+    Phoenix.View.render(PointingPartyWeb.CardView, "index.html", assigns)
+  end
+
+  def mount(%{username: username}, socket) do
+    Endpoint.subscribe("users")
```

Comment:
Can we go with a different topic name? What about `"pointing_party"` or something like that? Could be a little confusing to call the topic `"users"` when the topic is being used to broadcast both user and "pointing"-related events.

Reply:
`"pointing_party"` works for me. I'll update the code.